app/mainAppWindow/index.js

```diff
@@ -53,20 +53,34 @@ exports.onAppReady = function onAppReady() {
 		window = null;
 	});
 
-	window.loadURL(config.url);
+	url = processArgs(process.argv)
+	window.loadURL( url ? url:config.url);
 
 	if (config.webDebug) {
 		window.openDevTools();
 	}
 }
 
-exports.onAppSecondInstance = function onAppSecondInstance(args) {
+exports.onAppSecondInstance = function onAppSecondInstance(event, args) {
 	console.log('second-instance started');
 	if (window) {
-		console.log('focusing on window');
-		if (window.isMinimized()) window.restore();
```

Comment:
We probably need to keep this and the next window.focus logic. Probably at the bottom of the function (after the if)

Reply:
I kept it in `else` after the `if`

Reply:
If I am not missing anything, this means that if you open a meeting link, and you have your app is in the background (minimised), it will load the link but not focus the page. Maybe the window.loadUrl does that for you... happy if it does!

Reply:
I did it over the line 83 this file, through `windows.show`. Can you take look at that?